src/location.c

```diff
@@ -532,8 +532,8 @@ handle_start_in_thread_func (GTask *task,
 
       if (g_str_equal (app_id, ""))
         {
-          title = g_strdup (_("Grant Access to Your Location?"));
-          subtitle = g_strdup (_("An application wants to use your location."));
+          title = g_strdup (_("Allow Access to Your Location?"));
```

Comment:
The latest [mockups](https://gitlab.gnome.org/Teams/Design/os-mockups/-/raw/master/portals/portals.png) dropped the `Your` word. We should drop it here too.